webapp/src/utils/jira_issue_metadata.test.tsx

```diff
@@ -293,6 +293,105 @@ describe('utils/jira_issue_metadata', () => {
         expect(actual[0].values).toEqual([{value: '10001', label: 'Admin only'}, {value: '10000', label: 'Everyone'}, {value: '10002', label: 'Staff'}]);
     });
 
+    test('getStatusField should return options for statuses for selected issue types only', () => {
+        const metadata: IssueMetadata = {
+            projects: [
+                {
+                    key: 'TEST',
+                    issuetypes: [],
+                },
+            ],
```

Comment:
Is it realistic to have a blank array for `issuetypes` here?

Reply:
@mickmister No, I don't think this will ever be blank. This particular field is not useful in the function we are testing here, so we kept it empty. I you want we can add some mock data or maybe just add a comment above it.

Reply:
We should try to have realistic test data I suppose. I recommend using the saved test data when possible

Reply:
@mickmister Updated the test files